metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/dao/WorkflowExecutionDao.java

```diff
@@ -788,6 +790,19 @@ public WorkflowExecution getByTaskExecution(Date startedDate, PluginType pluginT
     return ExternalRequestUtil.retryableExternalRequestConnectionReset(query::first);
   }
 
+  public WorkflowExecution getByXsltId(String xsltId) {
```

Comment:
Maybe rename to 'getAnyWorkflowExecutionForXsltId'? Or 'getAnyByXsltId'? There is not just one execution for a xslt in general.

Reply:
ok